.github/workflows/sql-release.yml

```diff
@@ -5,6 +5,11 @@ on:
     paths:
       - ".github/workflows/sql-release.yml"
       - "sql/**"
+  pull_request:
+    paths:
+      - "sql/**"
+      - ".github/workflows/sql-release.yml"
```

Comment:
this will actually upload artifacts to gcs etc. do we want to do it for each PR. seems redundant to do it. We can just add a step to .travis.yaml which does `mvn -f sql/ -Pnative-lib -DskipTests` similar to https://github.com/rilldata/rill-developer/blob/main/.travis.yml#L73

Reply:
fixed, I removed uploading